reana-ui/src/pages/workflowDetails/components/WorkflowFiles.js

```diff
@@ -79,12 +81,54 @@ const PREVIEW_MIME_PREFIX_WHITELIST = {
   },
   "application/pdf": {
     serverPreviewable: true,
-    display: (content) => {
-      return <object data={content} className={styles["pdf-object"]} />;
-    },
+    display: (content, name) => (
+      <object data={content} className={styles["pdf-object"]}>
+        <a href={content} target="_blank" rel="noopener noreferrer">
```

Comment:
This link should work as a fallback in case the browser is not able to display the PDF file, right? If that's the case, what do you think about displaying a `Message` component (just like is done with other non-previewable types of files) instead of just the link?